django/common/scripts/cleaning/__init__.py

```diff
@@ -25,4 +26,4 @@
 from .map_status import map_status
 from .string_to_bool import string_to_bool
 from .strip import strip
-from .zero_to_empty import zero_to_empty
+from .zero_to_empty import zero_to_empty
```

Comment:
Have you installed pre-commit? I'm not sure this empty line at the end of the file should have been removed

Reply:
pre-commit params are moved in fhir-river ? I asked the question there and got no answers: https://github.com/arkhn/fhir-river/pull/512#discussion_r683590322

Reply:
Answer:
just install pre-commit
```
pip install pre-commit
pre-commit install
```